ssn/ssn_separated/ssn.ttl

```diff
@@ -663,35 +580,14 @@ Observation here records a Situation (the estimation of the value of a Property)
 
 skos:closeMatch 'measurement result' [VIM 2.9] http://www.bipm.org/utils/common/documents/jcgm/JCGM_200_2008.pdf
 
-Measurement result in VIM is the measured value plus any other relevant information, which means that measurement result and observation will often be associated to the same data (a value, a time, a property, etc.).""" ;
-                rdfs:comment "An Observation is a Situation in which a Sensing method has been used to estimate or calculate a value of a Property of a FeatureOfInterest.  Links to Sensing and Sensor describe what made the Observation and how; links to Property and Feature detail what was sensed; the result is the output of a Sensor; other metadata details times etc." ;
-                rdfs:isDefinedBy "http://www.w3.org/ns/ssn" ;
-                rdfs:label "Observation" ;
-                rdfs:seeAlso "http://www.w3.org/2005/Incubator/ssn/wiki/SSN_Skeleton#Skeleton" .
-
+Measurement result in VIM is the measured value plus any other relevant information, which means that measurement result and observation will often be associated to the same data (a value, a time, a property, etc.).""" .
 
-###  http://www.w3.org/ns/ssn/ObservationValue
-ssn:ObservationValue rdf:type owl:Class ;
-                     dc:source """skos:exactMatch 'measured quantity value' [VIM 2.10]
-                                  http://www.bipm.org/utils/common/documents/jcgm/JCGM_200_2008.pdf
-
-skos:exactMatch 'observed value' [SensorML OGC-0700]
-http://www.opengeospatial.org/standards/sensorml 
-
-skos:closeMatch 'observation result' [O&M]
-http://www.opengeospatial.org/standards/om 
-
-O&M conflates what we have as SensorOutput and ObservationValue into observation result, though the OGC standard does say \"result contains a value\" and \"a result which has a value\", which fits naturally with the model here.""" ;
-                     rdfs:comment "The value of the result of an Observation.  An Observation has a result which is the output of some sensor, the result is an information object that encodes some value for a Feature." ;
-                     rdfs:isDefinedBy "http://www.w3.org/ns/ssn" ;
-                     rdfs:label "Observation Value" ;
-                     rdfs:seeAlso "http://www.w3.org/2005/Incubator/ssn/wiki/SSN_Base#Data" .
 
 
 ###  http://www.w3.org/ns/ssn/OperatingPowerRange
 ssn:OperatingPowerRange rdf:type owl:Class ;
                         rdfs:subClassOf ssn:OperatingProperty ;
-                        rdfs:comment "Power range in which system or sensor is expected to operate." ;
+                        rdfs:comment "Power range in which a System, Sensor or Actuator is expected to operate." ;
```

Comment:
Power range in which a System is expected to operate.

(can apply to any system)